### DISCUSSION TITLE
only allow squash merges

### DISCUSSION THREAD

**Discussion by scarf005** [cataclysmbn/Cataclysm-BN#2440] [General]
![image](https://user-images.githubusercontent.com/54838975/226115702-208384e1-e44e-4c11-829f-0df0bc45d164.png)

by disabling merge commits and rebase merging, we could use squash merges which keeps commit history clean.

**Comment by Coolthulhu** [comment]
Sometimes squash merging is not possible to be done directly, but those cases can be solved locally, so it may be a good idea to disable other types.

**Reply by scarf005** [reply]
![image](https://user-images.githubusercontent.com/54838975/226229791-b528fe86-8ea5-4286-9649-5afbd2e414eb.png)

alright. disallowed merge and rebase merges. please let me know if there's any trouble. the rule's just for general convenience and not strict, therefore could be lifted if needed.
